code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -253,19 +253,29 @@
 		for(var/obj/item/bodypart/limb in C.get_damaged_bodyparts(TRUE, TRUE, FALSE, BODYPART_ROBOTIC))
 			robotic_limb_damage += limb.get_damage(stamina=FALSE)
 
+	var/datum/gas_mixture/air1 = airs[1]
+
 	if(mob_occupant.health >= mob_occupant.getMaxHealth() - robotic_limb_damage) // Don't bother with fully healed people. Now takes robotic limbs into account.
-		var/has_cryo_wound = FALSE
+		var/has_wound = FALSE
 		if(C && C.all_wounds)
-			for(var/datum/wound/wound as anything in C.all_wounds)
-				if(wound.wound_flags & ACCEPTS_CRYO)
+			if(air1.total_moles() && air1.get_moles(GAS_HEALIUM) > MINIMUM_MOLE_COUNT)
+				if(C.all_wounds)
```

Comment:
Redundant check for `C.all_wounds`? Was just checked two lines ago.

Reply:
Didn't realize lmao

Reply:
Imma fix this tmr